### DISCUSSION TITLE
Need some help with Rf Bridge with Tasmota and Portisch Fw

### DISCUSSION THREAD

**Discussion by drorya1** [arendst/Tasmota#12746] [Q&A] [answered]
Hello ,
I have a RF bridge with Tasmota and Portisch firmware where I try to control a skylight window.
While being successful for few occasions  , most of the times it just don't do anything .
While analyzing the protocol is seems that code is identical while what changes are bucket times.
When trying to control a RF ceiling fan I can trigger it off and on but it takes few attempts till it is working .
I wonder if there's any ability to better debug if my unit is defective ?
Might it be that a different configuration is needed ?
Can it be that code is too long ?
Any help will be highly appreciated 
In the bottom you can see some codes where I do rfraw 177 and press the RF remote for ~3 secs 

15:02:43 CMD: Group 0, Index 1, Command "RFRAW", Data "177"
15:02:43 RSL: stat/tasmota_5AFB0E/RESULT = {"RfRaw":"ON"}
15:02:43 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:43","RfRaw":{"Data":"AAA055"}}
15:02:45 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:45","RfRaw":{"Data":"AA B1 04 0064 00AA 01AE 01C2 381A28 55"}}
15:02:46 WIF: Checking connection...
15:02:51 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:51","RfRaw":{"Data":"AA B1 03 0154 00AA 0208 281919 55"}}
15:02:51 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:51","RfRaw":{"Data":"AA B1 03 010E 01F4 03C0 28181918 55"}}
15:02:52 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:52","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 0302 6022 48180818181818181818181808181818080808080808082808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:52 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:52","RfRaw":{"Data":"AA B1 03 0190 032A 032A A8090818 55"}}
15:02:52 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:52","RfRaw":{"Data":"AA B1 05 01CC 012C 0F0A 0302 5FF0 48181818181818181818080808080808080808080808082808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:52 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:52","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:52 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:52","RfRaw":{"Data":"AA B1 05 01E0 012C 0F00 0348 5FFA 48181818181818181818181818181818181818181818182818081B18081B080808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:52 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:52","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:53 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:53","RfRaw":{"Data":"AA B1 05 01D6 0122 0F00 02F8 5FFA 48181818181818181818181818181818180818180808082818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:53 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:53","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:53 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:53","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 5FF0 48181818181818181818181818181818181818181808182818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:53 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:53","RfRaw":{"Data":"AA B1 03 019A 0334 032A A8090818 55"}}
15:02:53 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:53","RfRaw":{"Data":"AA B1 05 01D6 0122 0F0A 02F8 6004 48181818181818181818181818181818181818180818182818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:53 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:53","RfRaw":{"Data":"AA B1 03 01A4 032A 0334 A8090818 55"}}
15:02:54 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:54","RfRaw":{"Data":"AA B1 05 01D6 0122 0F0A 02EE 6004 48181818181818181818181818181818181808180818182808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:54 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:54","RfRaw":{"Data":"AA B1 03 019A 0334 032A A8090818 55"}}
15:02:54 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:54","RfRaw":{"Data":"AA B1 05 01E0 012C 0F0A 0348 6004 48181818181818181818181818181818181818181818182818081B18080B080808080B080808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:54 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:54","RfRaw":{"Data":"AA B1 03 019A 032A 0334 A8090818 55"}}
15:02:54 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:54","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 6004 48181818181818181818181818181818181818081818082818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:54 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:54","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:54 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:54","RfRaw":{"Data":"AA B1 05 01E0 012C 0F00 0348 6004 48181818181818181818181818181818181818181818182818081B08081B180808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:55 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:54","RfRaw":{"Data":"AA B1 03 01A4 032A 0334 A8090818 55"}}
15:02:55 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:55","RfRaw":{"Data":"AA B1 05 01E0 012C 0F0A 0352 6004 48181818181818181818181818181818181818181818182818081B18081B080818080B183808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:55 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:55","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:55 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:55","RfRaw":{"Data":"AA B1 05 01D6 012C 0F00 02F8 5FFA 48181818181818181818181818181818081818081818082818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:55 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:55","RfRaw":{"Data":"AA B1 03 01A4 032A 0334 A8090818 55"}}
15:02:55 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:55","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02EE 6004 48181818181818181818181818181818181818181808182818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:55 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:55","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:56 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:56","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 6004 48181818181818181818181818181818181818081818082818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:56 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:56","RfRaw":{"Data":"AA B1 03 019A 032A 0334 A8090818 55"}}
15:02:56 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:56","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 6004 48181818181818181818181818181818181818180818082808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:56 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:56","RfRaw":{"Data":"AA B1 03 01A4 032A 0334 A8090818 55"}}
15:02:56 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:56","RfRaw":{"Data":"AA B1 05 01D6 0122 0F0A 02F8 600E 48181818181818181818181818181808181808181808182808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:56 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:56","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:57 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:57","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 600E 48181818181818181818181818081818181818081808082818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:57 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:57","RfRaw":{"Data":"AA B1 03 019A 032A 032A A8090818 55"}}
15:02:57 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:57","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02EE 6004 48181818181818181818181818181818180818180818182818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:57 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:57","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:57 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:57","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 6004 48181818181818181818181818181818181808081808082808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:57 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:57","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:57 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:57","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02EE 6004 48181818181818181818181818181818081818180818182808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:57 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:57","RfRaw":{"Data":"AA B1 03 019A 032A 0334 A8090818 55"}}
15:02:58 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:58","RfRaw":{"Data":"AA B1 05 01E0 012C 0F0A 0348 600E 48181818181818181818181818181818181818181818182818080B18080B080818080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:58 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:58","RfRaw":{"Data":"AA B1 03 019A 032A 0334 A8090818 55"}}
15:02:58 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:58","RfRaw":{"Data":"AA B1 05 01E0 012C 0F0A 0348 600E 48181818181818181818181818181818181818181818182818080B18080B080808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:58 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:58","RfRaw":{"Data":"AA B1 03 019A 0334 032A A8090818 55"}}
15:02:58 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:58","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 6004 48181818181818181818181818181818181808181808082808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:58 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:58","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:59 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:59","RfRaw":{"Data":"AA B1 05 01E0 012C 0F0A 0348 600E 48181818181818181818181818181818181818181818182818081B18081B080808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:59 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:59","RfRaw":{"Data":"AA B1 03 019A 032A 0334 A8090818 55"}}
15:02:59 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:59","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02EE 600E 48181818181818181818181818181818180818180818182808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:59 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:59","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:02:59 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:59","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 600E 48181818181818181818181818181818181818081818082818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:59 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:59","RfRaw":{"Data":"AA B1 03 019A 032A 0334 A8090818 55"}}
15:02:59 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:59","RfRaw":{"Data":"AA B1 05 01E0 012C 0F0A 0348 600E 48181818181818181818181818181818181818181818182818081B08081B080818080B183808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:02:59 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:02:59","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:03:00 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:00","RfRaw":{"Data":"AA B1 05 01E0 012C 0F0A 0352 600E 48181818181818181818181818181818181818181818182818081B08081B083808080B080808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:03:00 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:00","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:03:00 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:00","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 600E 48181818181818181818181818181808181818081818082818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:03:00 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:00","RfRaw":{"Data":"AA B1 03 01A4 032A 032A A8090818 55"}}
15:03:00 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:00","RfRaw":{"Data":"AA B1 05 01D6 0122 0F0A 02F8 600E 48181818181818181818181818181818181818181808082818380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:03:00 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:00","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:03:01 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:01","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 600E 48181818181818181818181818181818181808181818082808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:03:01 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:01","RfRaw":{"Data":"AA B1 03 019A 032A 0334 A8090818 55"}}
15:03:01 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:01","RfRaw":{"Data":"AA B1 05 01D6 012C 0F0A 02F8 600E 48181818181818181818181818181818181808081808182808380B08380B083808080B083808080B0838080B083B08083808080B3B08083B380B080838080B083B08080808380B08380B 55"}}
15:03:01 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:01","RfRaw":{"Data":"AA B1 03 019A 0334 0334 A8090818 55"}}
15:03:02 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:02","RfRaw":{"Data":"AA B1 03 00BE 0078 0212 280918 55"}}
15:03:04 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:04","RfRaw":{"Data":"AA B1 04 00BE 0172 006E 0226 381828 55"}}
15:03:06 RSL: tele/tasmota_5AFB0E/RESULT = {"Time":"2021-07-24T15:03:06","RfRaw":{"Data":"AA B1 03 00BE 0064 0230 280818 55"}}

**Comment by ascillato2** [comment, marked as answer]
As you didn't have any answer here, please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). 

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
